test/spanningtrees/kruskal.jl

```diff
@@ -40,3 +40,68 @@ push!(vec_mst, Edge(3, 4))
 push!(vec_mst, Edge(2, 3))
 
 @test mst == vec_mst
+
+#second test
+g2 = Graph(8)
```

Comment:
we are good, I'll add `minimum_spanning_tree` and some doc fixes in a later PR. Thanks!